### PR TITLE
Decrease minimal react version ( closed #84 )

### LINKED ISSUE #84: Problem with strict peerDependency
Firstly, thanks for this component, it's great.

Why this component has strict peerDependency react `15.6.1`?
Have you real causes for it?
Could it be changed to somthing like `>=15.4.0`?
We cannot use your component if we have other react version.
We very disappointed about it.

Thanks.

### PR DESCRIPTION
**Pull request by pofigizm**
All cli tests passed.
Hand tests is ok.
- Chrome, Firefox, Safari (mac)
- IE 11 (win7 in VirtualBox)

I did't change `react-dom` version, because `react-dom/test-utils` used in tests (since 15.5)
But I made hand tests with `react-dom 15.4.0`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,13 +64,13 @@
     "flow-copy-source": "1.2.1",
     "jest": "20.0.4",
     "raf-stub": "2.0.1",
-    "react": "15.6.1",
```
Comment (alexreardon) [inline package.json]: Can we please keep the dev dependency version the same and just change the peerDependency version?

Comment (pofigizm) [reply package.json]: It's good practice to use same version in develop. So you can quickly find breaking changes. But, it isn't problem for me. I've done it.

**Comment by pofigizm** [comment]
Hi, @alexreardon.
Thank you for approve.
Are you planning to merge it? How soon will it be released?

**Comment by alexreardon** [comment]
I am deep in work getting #2 finished so i have a bit of tunnel vision. I will be doing a release soon

**Pull request merged**